packages/discovery-provider/src/tasks/entity_manager/entity_manager.py

```diff
@@ -571,20 +586,27 @@ def collect_entities_to_fetch(update_task, entity_manager_txs):
 
             if entity_type == EntityType.TRACK:
                 entities_to_fetch[EntityType.TRACK_ROUTE].add(entity_id)
+                if action == Action.MUTE or action == Action.UNMUTE:
+                    entities_to_fetch[EntityType.COMMENT_NOTIFICATION_SETTING].add(
+                        (user_id, entity_id, entity_type)
+                    )
             if entity_type == EntityType.PLAYLIST:
                 entities_to_fetch[EntityType.PLAYLIST_ROUTE].add(entity_id)
             if entity_type == EntityType.COMMENT:
-                entities_to_fetch[EntityType.COMMENT].add(entity_id)
```

Comment:
Not sure I follow, whats the context removing this here?

Reply:
great catch! so this is actually unecessary because we do an `entities_to_fetch[entity_type].add(entity_id)` which catches comment add already